### DISCUSSION TITLE
Is there a way to auto load form data every x ms?

### DISCUSSION THREAD

**Discussion by cobraprojects** [filamentphp/filament#4317] [Help] [answered]
I have a model that receive data from an API (status)

I need to watch the status and change the form according to the this status

I just need to auto refresh (rerender)  the form every x seconds to get the new status from the database

I tried  `->extraAttributes(['wire:poll.visible.6000ms' => ''])` to my `Card` with no luck, although it rerender the component but data doesn't change

any ideas?

thanks in advabnce

**Comment by hrvoj3e** [comment]
Can you share how did you implement "I have a model that receive data from an API (status)". This is an Eloquent model or something different?

**Comment by cobraprojects** [comment, marked as answer]
I solved it with alpine js
```
->extraAttributes([
       'x-init' => 'setInterval(() => { $wire.afterFill() }, 10000);',
   ])
```
